src/sqs/sqs.ts

```diff
@@ -256,7 +257,10 @@ export class SqsQueue<MessagePayload> {
 				messageId: message.MessageId,
 				executionId: uuidv4(),
 				queue: this.queueName,
-				...(payload?.jiraHost ? { jiraHost: payload.jiraHost } : { })
+				jiraHost: payload?.jiraHost,
+				installationId: payload?.installationId,
+				gitHubAppId: payload?.gitHubAppConfig?.gitHubAppId,
+				webhookId: payload?.webhookId
```

Comment:
nit would these be visible in splunk as "jiraHost: undefined", are we fine with that?

Reply:
Good question, I think all msg in our queue should have jiraHost. 
And even if it is undefined, is there any concern on it?

Reply:
Close this, as undefined is "better"? As it is more explicit what's going? Lmk if you want to reopen it.